main.py

```diff
@@ -1,2 +1,279 @@
+# 1. List
+difList = ["string", 5, 10.5, False, True]
+print(difList)
+print(difList[1:-1])
+print(len(difList))
+
+difList.append("added")
+print(difList)
+
+difList.insert(1,123)
+print(difList)
+
+difList[2] = "changed"
+print(difList)
+
+difList[2:4] = ["inserted"]
+print(difList)
+
+difList.remove("inserted")
+print(difList)
+
+difList.pop(2)
+print(difList)
+
+# 2. List
+firstList = ["first", 2, 12.23]
+secondList = [False, True]
+
+firstList.append(secondList)  #second list was inserted  into first like separate list
+print(firstList)
+
+firstList.extend(secondList)  #second list was added to the end like another elements
+print(firstList)
+
+# 3. Dictionary
+diction = {
+    "name":"John",
+    "surename":"Deere",
+    "age": 25
+}
+print(diction["name"],diction["age"])
+print(diction.keys())
+print(diction.values())
+
+diction["age"] = 33
+print(diction.values())
+
+diction["maritalStatus"] = "married"
+print(diction["name"], diction["maritalStatus"])
+
+diction["children"] = ["Paul", "Jack", "Freddie"]
+print(diction["name"], diction["children"])
+
+del diction["age"]  # 1st method
+
+diction.pop("age")  # 2nd method
+print(diction)
+
+# 4. Tuples
+tuple = ("first","second", "third", 1, 2, 3, True, False, 10.1, 11.2, 12.3)
+print(tuple)
+print(tuple[-4:])
+print(tuple.index("third"))
+
+anotherTuple = ("next", "super", "best")
+print(tuple+anotherTuple)
+
+(one, two, three, *four) = tuple
+print(one)
+print(two)
+print(three)
+print(four)
+
+#LOOPSLOOPSLOOPSLOOPSLOOPSLOOPSLOOPSLOOPSLOOPSLOOPSLOOPSLOOPSLOOPSLOOPSLOOPSLOOPSLOOPSLOOPSLOOPS
+# 1.
+loopList = ["orange", "apple", "banana", "cherry"]
+for x in loopList:
+    print(x)
+
+for x in loopList:
+    if x=="apple":
+        print(x)
+
+for x in loopList:
+    print(x)
+    print(len(x))
+
+# 2.
+for x in range(2,100):
+    print(x)
+
+# 3.
+m = 0
+while m < 16:
+    print(m)
+    m = m+1
+
+# 4.
+n = 3
+while n < 34:
+    print(n)
+    n = n+1
+
+# 5.
+loopListList = ["orange", "apple", "banana", "cherry"]
+def litPrint():
+    for x in loopListList:
+        print(x)
+        if x=="banana":
+            break
+litPrint()
+
+def litPrint2():
+    for x in loopListList:
+        if x!="banana":
+            print(x)
+
+litPrint2()
+
+
+#FUNCTIONSFUNCTIONSFUNCTIONSFUNCTIONSFUNCTIONSFUNCTIONSFUNCTIONSFUNCTIONSFUNCTIONSFUNCTIONSFUNCTIONS
+# 1.
+def privet():
+    print("Hello, world")
+privet()
+
+# 2.
+def nameSurname(name,surname):
+    text1 ="Hello, my name is {} {}"
+    print(text1.format(name, surname))
+
+nameSurname("Peter", "Jackson")
+
+nameSurname(surname="Peterson", name="Jack")
+
+# 3.
+def models(*models):
+    print("The first car models is",models[0])
+
+models(1,2,3,4,5,6,7,8,9)
+
+# 4.
+def person(**personal):
+    print(personal["age"],personal["name"])
+
+person(surname="Doe", age=45, cat=True, name="John")
+
+# 5.
+def print_person_info(name, surname, maritalStatus="unknown",*age):
```

Comment:
```suggestion
def print_person_info(name, surname, marital_status="unknown",*age):
```
Use underscore '_' as separator in variables names